core/arch/arm/sm/std_smc.c

```diff
@@ -41,6 +41,67 @@ static const TEE_UUID uuid = {
 	{0x98, 0xd2, 0x74, 0xf4, 0x38, 0x27, 0x98, 0xbb},
 };
 
+#include <trace.h>
+
+static unsigned long arm_arch_version(void)
+{
+	return SMCCC_V_1_1;
```

Comment:
Why not go straight for v1.2?

Reply:
i'll check v1.2.

Reply:
v1.2 mandates a 31bit SoC ID for `SMCCC_ARCH_SOC_ID`. I wonder which values platforms should use.